app/coffee/LoggerSerializers.coffee

```diff
@@ -1,10 +1,23 @@
+_ = require('lodash')
+
 showLength = (thing) ->
-	"length: #{thing?.length}"
+	if thing?.length
+		"length: #{thing.length}"
+	else
+		thing
+
+showUpdateLength = (update) ->
+	if update?.op?.length
+		copy = _.cloneDeep(update)
```

Comment:
`clone` should be enough because `op` is at the top-level, and will avoid copying all the content of the ops.  In future maybe we can preserve some of the op content like the character positions `p:` and just replace `i:` and `d:` strings with their lengths so we can debug bad ops.

Reply:
That sounds like a good idea. I've updated the serializer to do that.